src/audio/tdfb/tdfb_direction.c

```diff
@@ -155,7 +157,7 @@ static bool line_array_mode_check(struct tdfb_comp_data *cd)
 
 	/* Cross product of vectors AB and AC is (0, 0, 0) if they are co-linear.
 	 * Form vector AB(a,b,c) from x(i+1) - x(i), y(i+1) - y(i), z(i+1) - z(i)
-	 * Form vector AC(d,e,f) from x(i+2) - x(i), y(i+2) - y(1), z(i+2) - z(i)
+	 * Form vector AC(d,e,f) from x(i+2) - x(i), y(i+2) - y(i), z(i+2) - z(i)
```

Comment:
Same, this part of code was changed in previous commit, would fit there better. The changes so far are initialize code changes (run once) and comments changes while the next one is good and actually reduces run-time MCPS.